src/E1_coldflow/E1_coldflow.cpp

```diff
@@ -169,7 +211,7 @@ void loop() {
       Automation::detectPeaks(loxInjector, propInjector);
     }
   }
-  delay(10);
+  delay(100);
```

Comment:
I assume this larger delay is just for testing.? & should be changed back before merging

Reply:
good catch